server/parameters/CoreConfigFactory.java

```diff
@@ -21,6 +21,7 @@
 import com.vaticle.typedb.common.yaml.Yaml;
 import com.vaticle.typedb.core.common.exception.TypeDBException;
 import com.vaticle.typedb.core.server.parameters.util.Option;
+import com.vaticle.typedb.core.server.parameters.util.YamlParser;
```

Comment:
All the changes in this file are from prefixing `Config`, `ConfigFactory` and `ConfigParser` classes with the word `Core`.